src/attr/converters.py

```diff
@@ -83,3 +86,117 @@ def default_if_none_converter(val):
             return default
 
     return default_if_none_converter
+
+
+def convert_attrs(cls):
```

Comment:
This is not our style of writing docstrings. 😇

(Check out https://github.com/python-attrs/attrs/blob/master/.github/CONTRIBUTING.rst#code – I know your style is the CPython one but I can't stand it because the rationale per PEP 257 is that Guido's Emacs can't format paragraphs otherwise 🤪).

Reply:
Alright, I will fix this. :)